core/variant/variant_call.cpp

```diff
@@ -968,8 +968,8 @@ static void _register_variant_builtin_methods() {
 
 	bind_method(String, to_int, sarray(), varray());
 	bind_method(String, to_float, sarray(), varray());
-	bind_method(String, hex_to_int, sarray("with_prefix"), varray(true));
-	bind_method(String, bin_to_int, sarray("with_prefix"), varray(true));
+	bind_method(String, hex_to_int, sarray(), varray(true));
+	bind_method(String, bin_to_int, sarray(), varray(true));
```

Comment:
The `true` should be removed, that's the default value for the last argument, which no longer exists.